x/group/keeper/genesis_test.go

```diff
@@ -44,14 +46,18 @@ var (
 
 func (s *GenesisTestSuite) SetupSuite() {
 	checkTx := false
-	db := dbm.NewMemDB()
-	encCdc := simapp.MakeTestEncodingConfig()
-	app := simapp.NewSimApp(log.NewNopLogger(), db, nil, true, map[int64]bool{}, simapp.DefaultNodeHome, 5, encCdc, simapp.EmptyAppOptions{})
+	var groupKeeper keeper.Keeper
+	app, err := simtestutil.Setup(
```

Comment:
```suggestion
	app, err := simtestutil.SetupAtGenesis(
```

This should fix the build :)

Reply:
thanks @julienrbrt